src/Extensions/Cassandra.AppMetrics/Cassandra.AppMetrics.csproj

```diff
@@ -36,8 +36,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="App.Metrics.Abstractions" Version="3.*" />
```

Comment:
These packages were pulling a dependency on .NET Standard 1 which includes packages with high severity vulnerability (NU1903) so I removed one and replaced the other.

Reply:
Hmm I understand the reasoning but apps are always free to update the version on the app side and if we want to upgrade the major version of App.Metrics I'd prefer if we do it on a separate PR

Reply:
Should I suppress the warning in the meantime?

Reply:
Well I don't have the warning anymore 😤